test/jdk/java/lang/Thread/virtual/HoldsLock.java

```diff
@@ -123,10 +125,10 @@ public void testThreadInfo() throws Exception {
 
             if (tid == carrierId) {
                 // Carrier is WAITING on vthread
-                assertEquals(info.getThreadState(), Thread.State.WAITING);
-                assertEquals(info.getLockInfo().getClassName(), vthread.getClass().getName());
-                assertEquals(info.getLockInfo().getIdentityHashCode(), System.identityHashCode(vthread));
-                assertEquals(info.getLockOwnerId(), vthreadId);
+                assertTrue(info.getThreadState() == Thread.State.WAITING);
+                assertEquals(vthread.getClass().getName(), info.getLockInfo().getClassName());
+                assertTrue(info.getLockInfo().getIdentityHashCode() == System.identityHashCode(vthread));
+                assertTrue(info.getLockOwnerId() == vthreadId);
```

Comment:
Previously, before the change to these lines, in case of failure, the `assertEquals` would have printed even the incorrect/unexpected value in the log file/exception. Now with the usage of `assertTrue`, that information is lost from the failures. Do you think, in the context of this test, it would be worth to continue using assertEquals and just switch the param order to match junit expectations.

Reply:
We will be coming back to this test in the future (part of it is disabled) so we can re-visit this test at that time.

Reply:
That sounds fine to me.